test/system/SmokeTestGame.cpp

```diff
@@ -153,6 +155,18 @@ BOOST_AUTO_TEST_CASE(host_server) {
             }
         }
 
+        if (m_current_turn == 1) {
+            // check home planet meters
+            bool found_planet = false;
+            for (const auto& planet : Objects().find<Planet>(OwnedVisitor(m_empire_id))) {
+                BOOST_REQUIRE_LT(0.0, planet->GetMeter(MeterType::METER_POPULATION)->Current());
+                BOOST_REQUIRE_LT(0.0, planet->GetMeter(MeterType::METER_INDUSTRY)->Current());
+                BOOST_REQUIRE_LT(0.0, planet->GetMeter(MeterType::METER_RESEARCH)->Current());
+                found_planet = true;
+            }
+            BOOST_REQUIRE(found_planet);
```

Comment:
Can these tests output something to the log when then pass?

Reply:
Yes, but it require `--log_level=message` command line argument